tox.ini

```diff
@@ -47,63 +69,15 @@ whitelist_externals =
 # in virtualenvs. Details: https://github.com/ansible/molecule/issues/1724
 sitepackages = true
 
-[testenv:lint]
-deps =
-    -rlint-requirements.txt
-extras =
-skip_install = true
-usedevelop = false
-
-[testenv:format]
-commands =
-    yapf -i -r molecule/ test/
-deps = yapf>=0.25.0,<2
-extras =
-skip_install = true
-usedevelop = false
-
-[testenv:format-check]
-commands =
-    yapf -d -r molecule/ test/
-deps = {[testenv:format]deps}
-extras = {[testenv:format]extras}
-skip_install = true
-usedevelop = false
-
-[testenv:doc]
+[doc]
+# basepython can't be put in the testenv section
```

Comment:
The reason why I do not support bumping tox too fast is because tox is a system requirement which means that if a new version of tox is released that breaks its use on one python based project and molecule is forcing me to use that breading-edge new version I may endup with a case where I cannot run tox with both projects.

To avoid this case we only need to be carefult to wait at least two weeks before using a newer version of tox. I am confident about tox is tested well but nobody can guarantee that a new release does not break anything.

The same issue does not apply for most other test requirements because they are installed inside virtualenvironments managed by tox, so no risk of conflict.

Shortly, lets bump it when we need new features but after a cooldown period so we avoid getting burned ;)